_pages/ato/checklist.md

```diff
@@ -38,9 +38,10 @@ Many of these tasks can be done in parallel...this is just a suggestion of prior
     * [ ] [Perform dynamic vulnerability scanning](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
         1. [ ] Resolve any visible security issues, re-running the scan as needed
         1. [ ] Add the issue-free scan report or [documentation about false positives](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/#caveats) to [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing)
-1. [ ] Update relevant documentation, primarily the README
+1. [ ] [Update relevant documentation](https://pages.18f.gov/before-you-ship/ato/tips/), primarily the README
 1. [ ] If this will be a new ATO, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to any currently-running sites
-1. [ ] Add a [System Security Plan](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
+1. [ ] Add a [System Security Plan yml](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
+1. [ ] Add a System Security Plan Google doc to [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing) (talk to your Infrastructure Lead for a template)
```

Comment:
Yes, for now. I wanted to get this requirement in as soon as possible for new ATO tickets being created.

Which template to use depends on which kind of ATO you are getting, which we don't talk about in this issue template at all. Rather than tackling all that, I'm pushing the work to the infra leads for now and we can update this template for multiple cases later.